### ISSUE TITLE
Biome encountered an unexpected error

### ISSUE THREAD

**Issue by RakhmonovShope** [biomejs/biome#7325]
### Environment information

```block
Source Location: crates/biome_service/src/settings.rs:78:14
Thread Name: tokio-runtime-worker
Message: You must have at least one workspace.
StdIO connection closed
Exit code: 0
Command line: /Users/rahmonovshohboz/.nvm/versions/node/v18.20.0/bin/node /Users/****/Documents/***/****/****/node_modules/@biomejs/biome/bin/biome lsp-proxy
```

### What happened?

After I change supported extensions it stopped formatting and linting

### Expected result

it should have formatted scss files also, but crashed

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by siketyan** [comment]
> it should have formatted scss files also, but crashed

Biome doesn't support parsing/formatting SCSS files.

**Comment by RakhmonovShope** [comment]
> > it should have formatted scss files also, but crashed
> 
> Biome doesn't support parsing/formatting SCSS files.

Thank you it worked after I removed .scss extenstion from config
